tests/trainers/test_training_loop.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+import unittest
+from unittest.mock import MagicMock
+
+import torch
+from omegaconf import OmegaConf
+
+from mmf.common.sample import SampleList
+from mmf.trainers.core.profiling import TrainerProfilingMixin
+from mmf.trainers.core.training_loop import TrainerTrainingLoopMixin
+
+DATA_ITEM_KEY = "test"
+
+
+class NumbersDataset(torch.utils.data.Dataset):
```

Comment:
It will make sense to move this to test utilities so it can be reused.

Reply:
Done.